### PR TITLE
feat(fp-351) branding component created

### PR DESCRIPTION
**Pull request by josue2591**
# Ticket(s)

- [FP-351: header logo](https://fourkitchens.clickup.com/t/36718269/FP-351)

## Notes
The #1 needs to be merged first

## Purpose
- Create the branding component
- In the [design](https://www.figma.com/file/wjvtxxU9UCb8Z1rM67ihYt/Assets?node-id=2%3A712&mode=dev) there are 4 "Logos" so this branding component was implemented to display any of those

### Functional Testing

- [x] Go to http://localhost:6006/?path=/story/elements-branding--default
- [ ] The branding component should look like the video shows
- [ ] It should allow to overwrite the href, the alt value and the image 

Video: https://www.loom.com/share/12bf158b79f248eca02a21935b6d2bc8?sid=4f63f918-f2c4-4072-a083-c0567b5cf271

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-351 Header logo](https://app.clickup.com/t/36718269/FP-351)

File: src/lib/components/02-components/branding/branding.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  src,
+  width,
+  height,
+  ...props
+}: BrandingProps) {
+  return (
+    <Link href={href} {...props}>
+      <img
+        src={src}
+        alt={`${alt}`}
```
Comment (cienvaras) [inline src/lib/components/02-components/branding/branding.tsx:23]: No need to use a template literal here.

```suggestion
        alt={alt}
```

**Review by cienvaras — CHANGES_REQUESTED** [review]
Looks great, just a little detail to fix.

**Review by mcortes19 — APPROVED** [review]
@josue2591 LGTM 🎖️

**Pull request merged**